CMakeLists.txt

```diff
@@ -308,7 +308,9 @@ if (KRATOS_SHARED_MEMORY_PARALLELIZATION STREQUAL "OpenMP")
 	if (OPENMP_FOUND)
```

Comment:
Despite the error the `add_definitions( -DKRATOS_SMP_OPENMP )` should be also inside this if (mainly to prevent people bypassing errors to still use omp.h).

Reply:
ah one can bypass those errors?
Didn't know, then I will make some more adaptions

Reply:
Sadly is possible if someone tampers with the policies in the cmd, should not be at the grasp of the end users but some systems I found that have policies pre-set when you load the cmake module (mainly clusters). So, just to be extra-safe

Reply:
now should be better